qiskit_experiments/randomized_benchmarking/interleaved_rb_analysis.py

```diff
@@ -37,11 +38,10 @@ class InterleavedRBAnalysis(RBAnalysis):
 
     The error bounds are given by
     :math:`E=\min\left\{ \begin{array}{c}
-    \frac{\left(d-1\right)\left[\left|p-p_{\overline{\mathcal{C}}}/p\right|+\left(1-p\right)\right]}{d}\\
+    \frac{\left(d-1\right)\left[\left|p-p_{\overline{\mathcal{C}}}\right|+\left(1-p\right)\right]}{d}\\
```

Comment:
Are the systematic error bounds calculation remain the same in the case?
Need to comment that the notation here is not the same as in the original interleaved RB paper, 
namely that p_C here denotes p_C/p

Reply:
I'm not 100% sure, I was hoping you or @gadial could double check this.

Reply:
It looks OK to me, but maybe it's better to ask one of the co-authors of the original paper?
(and one should also comment that the notation is different than the original paper)